people/tests.py

```diff
@@ -95,12 +95,12 @@ def test_list_of_contributors_contains_all_live_contributors(self):
         six.assertCountEqual(
             self,
             all_live_contributors,
-            contributor_list.contributors
+            contributor_list.nonfeatured_contributors
         )
 
     def test_list_of_contributors_sorted_by_last_name(self):
         contributor_list = models.ContributorListPage.objects.all().first()
-        contributors = contributor_list.contributors
+        contributors = contributor_list.nonfeatured_contributors
```

Comment:
On line 110 of this file, need to account for rows:
self.assertSequenceEqual(contributors, [[joe, beth, bob, mary],])